content/en/real_user_monitoring/error_tracking/mobile/expo.md

```diff
@@ -80,15 +68,33 @@ Run `eas secret:create` to set `DATADOG_SITE` to the host of your Datadog site,
 | `androidProguardMappingFiles` | `true`  | Enables the uploading of Proguard mapping files to deobfuscate native Android crashes (is only applied if obfuscation is enabled). |
 | `datadogGradlePluginVersion`  | `"1.+"` | Version of `dd-sdk-android-gradle-plugin` used for uploading Proguard mapping files.     |
 
-### Limitations
+## Get deobfuscated stack traces
+
+### Add git repository data to your mapping files on Expo Application Services (EAS)
+
+If you are using EAS to build your Expo application, set `cli.requireCommit` to `true` in your `eas.json` file to add git repository data to your mapping files.
+
+```json
+{
+    "cli": {
+        "requireCommit": true
+    }
+}
+```
+
+## Limitations
 
 {{< site-region region="us,us3,us5,eu,gov" >}}
 Source maps, mapping files, and dSYM files are limited to **500** MB each.
 {{< /site-region >}}
 {{< site-region region="ap1" >}}
-Source maps, mapping files, and dSYM files  are limited to **500** MB each.
+Source maps, mapping files, and dSYM files are limited to **500** MB each.
 {{< /site-region >}}
 
+## Test your implementation
+
+## Additional configuration options
```

Comment:
Back to back H2 headers, is this intentional?

Reply:
Yes it is intentional but the Test your implementation section shouldn't be empty. Thanks for calling this out, I'll need to get it filled in!